src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,8 +1,23 @@
 package mate.academy.spring.dto.request;
 
+import java.util.Objects;
+import javax.validation.Valid;
+import javax.validation.constraints.AssertTrue;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import mate.academy.spring.model.validation.Email;
+
 public class UserRequestDto {
+    @Email
+    @NotNull
+    @NotBlank
     private String email;
```

Comment:
```suggestion
    @Email
    private String email;
```

<img width="1110" alt="image" src="https://github.com/mate-academy/jv-spring-security/assets/81781479/0d9b637d-e21d-44d1-9d61-78f77fb6e9a3">